### ISSUE TITLE
2 CATTLE_NETWORK_POLICY forward chain present with network policy is enabled.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#9987]
Rancher Server version - Build from v1.6
rancher/network-manager:v0.7.10 and rancher/net:v0.13.0

Enable network policy manager and set network policy to allow traffic only within services.
iptables forward chain shows , 2 CATTLE_NETWORK_POLICY forward chain , one for 0.0.0.0/0 and one for 10.42 subnet.

```
root@sangeev16-10acre-1:/home/sangeethahariharan1# iptables -L FORWARD -n -v
Chain FORWARD (policy DROP 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
 265K 1426M CATTLE_NETWORK_POLICY  all  --  *      *       0.0.0.0/0            0.0.0.0/0           
 265K 1426M CATTLE_FORWARD  all  --  *      *       0.0.0.0/0            0.0.0.0/0           
 130K   77M DOCKER-ISOLATION  all  --  *      *       0.0.0.0/0            0.0.0.0/0           
93204   71M DOCKER     all  --  *      docker0  0.0.0.0/0            0.0.0.0/0           
93200   71M ACCEPT     all  --  *      docker0  0.0.0.0/0            0.0.0.0/0            ctstate RELATED,ESTABLISHED
36732 5615K ACCEPT     all  --  docker0 !docker0  0.0.0.0/0            0.0.0.0/0           
    4   240 ACCEPT     all  --  docker0 docker0  0.0.0.0/0            0.0.0.0/0           
  530 32239 CATTLE_NETWORK_POLICY  all  --  *      *       10.42.0.0/16         10.42.0.0/16        
root@sangeev16-10acre-1:/home/sangeethahariharan1# 
```

**Comment by leodotcloud** [comment]
Fixed with`network-manager:v0.7.11`

**Comment by sangeethah** [comment]
Rancher Server version - v1.6.11-rc2
rancher/network-manager:v0.7.11 and rancher/net:v0.13.0

Enable network policy manager and set network policy to allow traffic only within services.
iptables forward chain shows only 1 CATTLE_NETWORK_POLICY forward chain for 10.42 subnet.
